server/public/offen-icon-black.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
As the `server` currently has no build pipeline in place, can we compress this SVG to be as small as possible?

Reply:
ill try